recipes/flit/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.9" %}
+
+package:
+  name: flit
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/takluyver/flit.git
+  git_tag: {{ version }}
```

Comment:
Sorry. I've been meaning to do that for so long that I ran out of excuses :flushed: